src/app/admin/feed/_components/ClubFeed.tsx

```
@@ -84,25 +84,39 @@ function FeedImageWithRadio({
       />
+      {feed.feedType === 'VIDEO' && (
+        <div className="absolute right-2 bottom-2 z-20 flex items-center justify-center rounded-full bg-black p-2">
+          <svg
+            width="20"
+            height="20"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="text-white"
+          >
+            <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)